resolver_wrapper.go

```diff
@@ -77,6 +77,7 @@ func (ccr *ccResolverWrapper) start() error {
 			CredsBundle:          ccr.cc.dopts.copts.CredsBundle,
 			Dialer:               ccr.cc.dopts.copts.Dialer,
 			Authority:            ccr.cc.authority,
+			MetricsRecorder:      ccr.cc.metricsRecorderList,
```

Comment:
Can you make sure the delegating resolver is properly forwarding?  It's the only resolver that should be doing this kind of thing:

https://pkg.go.dev/google.golang.org/grpc@master/internal/resolver/delegatingresolver

Reply:
Yup takes the opts provided and passes it to targetResolverBuilder.Build(target, cc, opts) on line 107.